FogCreek/Glitch-Community
# Innate tractor: Componentize progress, input styles

## Links
* Remix link: http://innate-tractor.glitch.me

## Changes:
* create Progress component
* copy default input styles onto TextInput

## How To Test:
* log into http://innate-tractor.glitch.me
* verify input styles are correct for top nav search, add project to collection search, editable profile fields
* verify progress bars render correctly in team analytics (used as bar chart in Referrers section)
